File: p2p/src/main/java/com/androdevlinux/percy/p2p/common/listeners/ServiceRegisteredListener.java
package com.androdevlinux.percy.p2p.common.listeners;


import com.androdevlinux.percy.p2p.common.WiFiP2PError;

public interface ServiceRegisteredListener {

    void onSuccessServiceRegistered();

    void onErrorServiceRegistered(WiFiP2PError wiFiP2PError);

}

File: p2p/src/main/java/com/androdevlinux/percy/p2p/common/listeners/ServiceRegisteredListener.kt
package com.androdevlinux.percy.p2p.common.listeners


import com.androdevlinux.percy.p2p.common.WiFiP2PError

interface ServiceRegisteredListener {

    fun onSuccessServiceRegistered()

    fun onErrorServiceRegistered(wiFiP2PError: WiFiP2PError)

}
